firebase/src/components/surgeryPage/surgeryPagePublicInfo.tsx

```diff
@@ -4,53 +4,147 @@ import {
 } from "@material-ui/core";
 import * as React from "react";
 import { useSelector } from "react-redux";
+import { IFirestoreSurgery } from "@mikoroltanak/api";
 import { getGlobalServices } from "../../services/services";
 import { selectSurgeries } from "../../store/selectors";
+import { surgeryValidator, useChangeHandler } from "./surgeryPageUtils";
 
 export function SurgeryPagePublicInfo({ surgeryId }: { surgeryId: string }) {
+
     const surgeries = useSelector(selectSurgeries);
-    const surgery = surgeries[surgeryId];
+    const surgery: IFirestoreSurgery = surgeries[surgeryId];
```

Comment:
Ez miért kell? Elvileg következik a típusból

Reply:
Igen kellene, hogy feldobja neki de mégis `any`-ként hivatkozott később rá a TS. Ha így megadtam a típust, akkor később is szépen látta a típust. De ha esetleg megy nálad local-ban nézz rá kérlek. Lehet nálam nem frankó valami.

Reply:
Nálam akkor szokta anyként kiírni, ha mondjuk újrafordítottam egy package-et, vagy valami hiba volt korábban. A surgeries típusát jól látja még a ts? És a useSelector típusát? selectSurgeries? Hol rontja el először?

Reply:
<img width="569" alt="image" src="https://user-images.githubusercontent.com/25322572/111885337-6c18e480-89c7-11eb-9960-90b705dbeea5.png">
Itt még látszik, hogy a selector a SurgeryState-el fog visszatérni, de úgy tűnik valamiért az nem esik le neki, hogy annak az elemei ezt az interface-t implementálják.
<img width="656" alt="image" src="https://user-images.githubusercontent.com/25322572/111885312-54416080-89c7-11eb-9d8d-5b5234a5c80e.png">

Reply:
UPDATE: csináltam egy restart-ot a VSCode-nak és megjavult.